wpengine/unibody
# Mar 5661 button styles

## Status
**READY**

## JIRA Ticket
[A link to the relevant JIRA ticket](https://wpengine.atlassian.net/browse/MAR-5661)

## Description
Minor updates to button styles. Also updates the styleguide to include all button variations and button groups. 

## Steps to Test or Reproduce
Build the styleguide, and check out the button and button group pages:
http://localhost:3000/components/detail/button-anchor--default (with links on the side)
http://localhost:3000/components/detail/button-group 

## Random GIF
Include one random GIF to "pay" your reviewers in humor.

![Thanks for reviewing my code!](https://media.giphy.com/media/11tRBTlIlmb10k/giphy.gif)
